tests/integration/test_interactive.py

```diff
@@ -95,3 +118,110 @@ def test_interactive_submission_success(rf, local_repo, enable_network):
     with tempfile.TemporaryDirectory(suffix=f"repo-{ar_pk}") as path:
         git("clone", repo.url, path)
         local_run.main(path, ["run_all"])
+
+
+def test_interactive_publishing_report_success(client, release, slack_messages):
+    # set up the project…
+    project = ProjectFactory()
+    workspace = WorkspaceFactory(project=project, name=project.interactive_slug)
+
+    # … and the user
+    user = UserFactory()
+    ProjectMembershipFactory(project=project, user=user, roles=[InteractiveReporter])
+    client.force_login(user)
+
+    # "run" the job
+    job_request = JobRequestFactory(workspace=workspace)
+    JobFactory(job_request=job_request, status="succeeded")
+
+    report = ReportFactory(
+        release_file=release.files.first(), project=project, title="My report title"
+    )
+    analysis = AnalysisRequestFactory(
+        job_request=job_request, project=project, report=report
+    )
+
+    # this is a complex stack of factories so this is a little sense check to
+    # ensure the analysis is in the expected success state to allow us to
+    # request it's published
+    assert analysis.status == "succeeded"
+
+    # user can view their analysis page and the report is rendered there
+    with patch.object(
+        AnalysisRequestCreate, "get_opencodelists_api", new=FakeOpenCodelistsAPI
+    ):
+        response = client.get(project.get_interactive_url())
+        assert response.status_code == 200
+        # TODO: check rendered response has the report details
```

Comment:
This TODO is confusing. Can this be implemented or removed, please?

Reply:
It referenced the commented out line below which I forgot to fix 😂

Reply:
And exposed a broken part of the test 🎉